orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -128,36 +131,34 @@ class RunTaskHandler(
   }
 
   private fun trackResult(stage: Stage, taskModel: com.netflix.spinnaker.orca.pipeline.model.Task, status: ExecutionStatus) {
-    val id = registry.createId("task.invocations")
-      .withTag("status", status.toString())
-      .withTag("executionType", stage.execution.type.name.capitalize())
-      .withTag("taskType", taskModel.implementingClass)
-      .withTag("isComplete", status.isComplete.toString())
+    val commonTags = arrayListOf(
+      BasicTag("status", status.toString()),
+      BasicTag("executionType", stage.execution.type.name.capitalize()),
+      BasicTag("isComplete", status.isComplete.toString()),
+      BasicTag("cloudProvider", stage.context["cloudProvider"].toString()?: "n/a"))
+
+    val taskInvocationsId = registry.createId("task.invocations")
+      .withTags(commonTags)
       .withTag("application", stage.execution.application)
-      .let { id ->
-        stage.context["cloudProvider"]?.let {
-          id.withTag("cloudProvider", it.toString())
-        } ?: id
-      }
-    registry.counter(id).increment()
 
-    val distributionId = registry.createId("task.invocations.duration").withTags(id.tags())
-    BucketCounter
-      .get(registry, distributionId, { v -> bucketDuration(v) })
-      .record(System.currentTimeMillis() - (taskModel.startTime ?: 0))
-  }
+    val taskInvocationsWithTypeId = registry.createId("task.invocations.withType")
+      .withTags(commonTags)
+      .withTag("taskType", taskModel.implementingClass)
+      .withTag("account", stage.context["account"].toString()?: "n/a")
```

Comment:
The `/` character is invalid and will be rewritten to an underscore `_`.

See http://insight-docs.prod.netflix.net/atlas/java-client/#allowed-characters for more details.